README.rst

```diff
@@ -1,83 +1,161 @@
+     tf.keras.layers.Dense(10, activation='softmax')
+   ])
+
+   model.compile(optimizer='adam',
+                 loss='sparse_categorical_crossentropy',
+                 metrics=['accuracy'])
+
+   model.fit(x_train, y_train, epochs=1)
+
+   model.evaluate(x_test, y_test)
+
+.. _header-n965:
+
+The Dockerfile
+~~~~~~~~~~~~~~
+
+We can create a Dockerfile with our dependencies and defining the
+program that will be executed in SageMaker:
+
+.. code:: docker
+
+   FROM tensorflow/tensorflow:2.0.0a0
+
+   RUN pip install sagemaker-containers
+
+   # Copies the training code inside the container
+   COPY train.py /opt/ml/code/train.py
 
-   # starts the training job with an input data channel named training pointing to
-   # s3://bucket/path/to/training/data
-   # for more information about data channels, see
-   # https://docs.aws.amazon.com/sagemaker/latest/dg/your-algorithms-training-algo.html#your-algorithms-training-algo-running-container-inputdataconfig
-   chainer_estimator.fit({'training': 's3://bucket/path/to/training/data', 'testing': 's3://bucket/path/to/testing/data')
+   # Defines train.py as script entrypoint
+   ENV SAGEMAKER_PROGRAM train.py
 
-For more on using the SageMaker Python SDK for interacting with the various frameworks, see their repsective documentation:
+.. _header-n968:
 
-- TensorFlow: https://github.com/aws/sagemaker-python-sdk#tensorflow-sagemaker-estimators
-- MXNet: https://sagemaker.readthedocs.io/en/stable/using_mxnet.html
-- PyTorch: https://github.com/aws/sagemaker-python-sdk#pytorch-sagemaker-estimators
-- Chainer: https://github.com/aws/sagemaker-python-sdk#chainer-sagemaker-estimators
+Building the container
+~~~~~~~~~~~~~~~~~~~~~~
+
+We can build the container like any docker container:
+
+.. code:: shell
+
+   docker build -t tf-2.0 .
+
+.. _header-n971:
+
+Training with Local Mode
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+We can use `Local
+Mode <https://sagemaker.readthedocs.io/en/stable/overview.html#local-mode>`__
+to test the container locally:
+
+.. code:: python
+
+   from sagemaker.estimator import Estimator
+
+   estimator = Estimator(image_name='tf-2.0',
+                         role='SageMakerRole',
+                         train_instance_count=1,
+                         train_instance_type='local')
+
+   estimator.fit()
+
+To see a complete example on how to create a container using SageMaker
+Container, including pushing it to ECR, see `Building your own
+container <https://github.com/awslabs/amazon-sagemaker-examples/blob/master/advanced_functionality/tensorflow_bring_your_own/tensorflow_bring_your_own.ipynb>`__.
+
+.. _header-n975:
 
 How a script is executed inside the container
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+---------------------------------------------
+
+The environment variable **SAGEMAKER_PROGRAM** determines the entrypoint
+name that will be executed during training. This script must be located
+under the folder **/opt/ml/model**. The following scripts are supported:
```

Comment:
* some of these bolded words should probably be put in backticks instead
* suggestion for the first two sentences: "The training script must be located under the folder `/opt/ml/model` and its relative path is defined in the environment variable `SAGEMAKER_PROGRAM`."